tilecloud_chain/generate.py

```diff
@@ -370,15 +349,14 @@ def _get_tilestore_and_message_for_layer(self, layer, gene):
                     params=params,
                 ),),
                 headers=layer['headers'],
-            ), "Get tile from WMS")
+            )
         elif layer['type'] == 'mapnik':  # pragma: no cover
             from tilecloud.store.mapnik_ import MapnikTileStore
             from tilecloud_chain.mapnik_ import MapnikDropActionTileStore
 
             grid = gene.get_grid(layer['grid'])
             if layer['output_format'] == 'grid':
-                self.count_tiles = gene.counter()
-                return (MapnikDropActionTileStore(
+                return MapnikDropActionTileStore(
```

Comment:
We should add a try catch block around this block to make the Travis tests pass...